docs/clients.md

```diff
@@ -107,6 +107,18 @@ val lbc = new LoadBalancingClient[Redis](worker, generator, 3, clients)
 Redis.client(lbc).zadd(ByteString("key"), ByteString("value"))
 {% endhighlight %}
 
+### Retry Policy
+
+Each service client takes in a retry policy. The client defaults to an 
```

Comment:
It should be a bit more clear in this section that the RetryPolicy is used for retrying to connect, not for retrying a request.  Also we should start linking more frequently to the scaladocs, for example, here are the docs for [RetryPolicy](https://tumblr.github.io/colossus/api/index.html#colossus.core.RetryPolicy) and [BackoffPolicy](https://tumblr.github.io/colossus/api/index.html#colossus.core.BackoffPolicy)